editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -130,57 +150,64 @@ export const MembersTable = ({
                   />
                 </TableCell>
                 <TableCell>{member.email}</TableCell>
-                {showEditMemberButton && (
-                  <Permission.IsPlatformAdmin>
-                    <TableCell>
+                <Permission.IsPlatformAdmin>
+                  <TableCell>
+                    {showEditMemberButton && (
                       <EditUserButton
                         onClick={() => {
-                          setShowUpdateModal(true);
+                          setActionType("edit");
+                          setShowAddModal(true);
                           setInitialValues(member);
                         }}
                         data-testId={`edit-button-${i}`}
                       >
                         Edit
                       </EditUserButton>
-                    </TableCell>
-                  </Permission.IsPlatformAdmin>
-                )}
+                    )}
+                  </TableCell>
+                </Permission.IsPlatformAdmin>
+                <Permission.IsPlatformAdmin>
+                  <TableCell>
+                    {showEditMemberButton && (
+                      <RemoveUserButton
+                        onClick={() => {
+                          setActionType("remove");
+                          setShowAddModal(true);
+                          setInitialValues(member);
+                        }}
+                        data-testId={`remove-button-${i}`}
+                      >
+                        Remove
+                      </RemoveUserButton>
+                    )}
+                  </TableCell>
+                </Permission.IsPlatformAdmin>
               </StyledTableRow>
             ))}
             {showAddMemberButton && (
-              <Permission.IsPlatformAdmin>
-                <TableRow>
-                  <TableCell colSpan={3}>
-                    <AddButton
-                      onClick={() => {
-                        setInitialValues(undefined);
-                        setShowAddModal(true);
-                      }}
-                    >
-                      Add a new editor
-                    </AddButton>
-                  </TableCell>
-                </TableRow>
-              </Permission.IsPlatformAdmin>
+              <TableRow>
+                <TableCell colSpan={5}>
+                  <AddButton
+                    onClick={() => {
+                      setActionType("add");
+                      setInitialValues(undefined);
+                      setShowAddModal(true);
+                    }}
+                  >
+                    Add a new editor
+                  </AddButton>
+                </TableCell>
+              </TableRow>
             )}
           </TableBody>
         </Table>
       </TableContainer>
       {showAddModal && (
-        <EditorUpsertModal
+        <SettingsModal
```

Comment:
`SettingsModal` has been used here to separate out the logic of what content goes inside a `<Dialog>` - felt right considering the 3 different options which contain two different types of dialog content, but all reusing similar props